openslides/core/static/templates/index.html

```diff
@@ -210,7 +210,7 @@
     </div>
 
     <!-- col1 -->
-    <div class="col1" ng-class="{
+    <div id="col1" class="col1" ng-class="{
```

Comment:
Can we please use a more explicit id (like sidebar for col2)? (main-column or something like this)